setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
+filename = *.py, *.pyx
+ignore = E501, W503
+per_file_ignores = gdist.pyx: E999, E402, E266, E255, E227, E226, E225
```

Comment:
Would it be possible to copy with every ignored error code also the explanation for it (what we are ignoring).
e.g. `E501 #  Line too long (82 > 79 characters)` ?

Reply:
Is  `E402 | Module level import not at top of file` ?
Usually I don't like the stile of an import in another place that the top, unless there is a good reason. Do we really need this exception here?

Reply:
Also are these the ignores we really want:
`E225 | Missing whitespace around operator
E226 | Missing whitespace around arithmetic operator
E227 | Missing whitespace around bitwise or shift operator`
They would seem fitting to be obeyed, to me.

Reply:
For `pyx` files, flake8 will complain for the pointers (treated as multiplication operator in python) and address-of operator (bitwise and operator in python).